journalwatch.py

```diff
@@ -364,15 +369,23 @@ def parse_config_files():
     return cfg, patterns
 
 
-def get_journal(since=None):
+def get_journal(since=None, priority=None):
     """Open the journal and get a journal reader.
 
     Args:
         since: A datetime object where to start reading.
                If None, the whole journal is read.
+        priority: A number between 1 and 7 for lowest priority
+               to consider.
+               If omited, defaults to "LOG_INFO", number 6.
     """
     j = journal.Reader()
-    j.log_level(journal.LOG_INFO)
+    if (int(priority) < 1):
```

Comment:
I included `type=int` in the argument parser line, which means that even with the "try" statement I get this as an error:
````
$journalwatch --priority "not-an-integer"
usage: journalwatch [-h] [--since [SINCE]] [--priority [PRIORITY]]
                    [--loglevel [LOGLEVEL]] [--mail_from [MAIL_FROM]]
                    [--mail_to [MAIL_TO]] [--mail_binary [MAIL_BINARY]]
                    [--mail_args [MAIL_ARGS]] [--mail_subject [MAIL_SUBJECT]]
                    [--version]
                    [ACTION]
journalwatch: error: argument --priority: invalid int value: 'not-an-integer'
````
If I remove that check from the arguments it goes so:
````
$journalwatch --priority "not-an-integer"
Priority level invalid: 'not-an-integer'
````
Guess both have their merits, but the second is more consistent (I notice you haven't used any type checking in the argument parser).